packages/web-components/src/components/va-alert/va-alert.css

```diff
@@ -45,6 +40,13 @@
   max-width: 1000px;
 }
 
+@media screen and (min-width:1024px) {
+  :host([full-width='true']) .va-alert-close,
+  :host([full-width='']) .va-alert-close {
+    position: relative;
+  }
+}
+
```

Comment:
Is this required or can we get rid of it?

Reply:
Ah, I see it's needed on large screens because the alert contents are centered for full-width alerts and `max-width`. This is required to keep the button inside the 1000px.

```css
:host([full-width='true']),
:host([full-width='']) {
  border-top-style: solid;
  border-top-width: 10px;
  display: flex;
  align-items: flex-start;
  justify-content: center;
}

:host([full-width='']) div.alert,
:host([full-width='true']) div.alert {
  border-left: none;
  max-width: 1000px;
}
```